### DISCUSSION TITLE
FrondEnd - MVC - ilustración

### DISCUSSION THREAD

**Discussion by RicardoFT1** [Launch-X-Latam/MisionFrontEnd#33] [Q&A]
¿Es posible actualizar la imagen del MVC? 

Tiene información con fuente de color negro, la mayoría usamos el GitHub con tema oscuro por tal motivo la información se pierde con el fondo.

Gracias.

**Comment by gersonjeanpierre** [comment]
Puedes usar esta opción hasta que modifiquen.

![image](https://user-images.githubusercontent.com/114420460/192567577-1a3a3086-f3cc-4efe-85c1-dba84f375df1.png)

**Comment by ayax-m** [comment]
> ¿Es posible actualizar la imagen del MVC? 
> 
> 
> 
> Tiene información con fuente de color negro, la mayoría usamos el GitHub con tema oscuro por tal motivo la información se pierde con el fondo.
> 
> 
> 
> Gracias.

¿Qué significa MVC?

**Reply by nahuelRo** [reply]
Es una arquitectura de software que significa modelo vista y controlador.

**Reply by JorgeDevmm** [reply]
Exacto bro

**Comment by doguedogue** [comment]
Tienes toda la razón, mete un PR 🚀

**Comment by Emilio17MC** [comment]
Hola!

Espero te sea útil

![image](https://user-images.githubusercontent.com/6946456/192631902-1aadf0ef-fa6b-4763-bfc8-48534f326a0e.png)

**Comment by RicardoFT1** [comment]
Hola, agradezco a todos la información que me comparte y espero que a los demás Explorer que pasaron por el capitulo mencionado les sirva el contenido que ustedes compartieron aquí, para mi ya quedo resuelto la información que no lograba visualizar, sin embargo seria bueno que ajustaran esa parte en el curso. @romarpla 

Cordialmente.

Ricardo F.
